2dlib/memsurf.cpp

```diff
@@ -267,7 +265,8 @@ void gr_mem_surf_Clear(ddgr_surface *dsf, ddgr_color col, int l, int t, int w, i
 bool gr_mem_surf_Blt(ddgr_surface *dsf, int dx, int dy, ddgr_surface *ssf, int sx, int sy, int sw, int sh) {
   //	Maybe we should allow slow 16bpp to 24bpp and vice-versa blts.
   //	for now, we wont.
-  mem_bitmap *sbm = (mem_bitmap *)ssf->obj, *dbm = (mem_bitmap *)dsf->obj;
+  [[maybe_unused]] mem_bitmap *sbm = (mem_bitmap *)ssf->obj; 
```

Comment:
`[[maybe_unused]]` because it is only used in the assertion? In that case, its value can directly be used in the `ASSERT` statement without initializing a variable

Reply:
True. I've changed that in the patch and will force push the rebased branch soon.

Reply:
thanks for fixing :)